gp-includes/install-upgrade.php

```diff
@@ -10,9 +21,21 @@ function gp_upgrade_db() {
 	update_option( 'gp_db_version', GP_DB_VERSION );
 }
 
+/**
+ * Updates existing data in the database during an upgrade.
+ *
+ * @since 1.0.0
+ *
+ * @param int $db_version The current version of the database before the upgrade.
+ */
 function gp_upgrade_data( $db_version ) {
 	global $wpdb;
+
 	if ( $db_version < 190 ) {
-		$wpdb->query("UPDATE $wpdb->gp_translations SET status = REPLACE(REPLACE(status, '-', ''), '+', '');");
+		$wpdb->query( "UPDATE {$wpdb->gp_translations} SET status = REPLACE(REPLACE(status, '-', ''), '+', '');" );
+	}
+
+	if ( $db_version < 950 ) {
+		$wpdb->query( "UPDATE {$wpdb->gp_projects} SET `path`=SUBSTR(`path`, 1, LENGTH(`path`) - 1) WHERE `path` LIKE '%/';" );
```

Comment:
This needs to be `CHAR_LENGTH()` and let's use `SUBSTRING()`. Some whitespace before/after the equal sign would be nice too.